src/components/MethodField.vue

```diff
@@ -65,22 +70,22 @@ const setValues = async () => {
     if (
         !props.trx.parsedTransaction
         && props.trx.from === ZERO_ADDRESSES
-        && props.trx.value
+        && propValue.value
         && parseInt(props.trx.gasPrice as string) === 0
     ) {
         nativeTooltipText.value = $t('pages.transactions.native_deposit_tooltip');
         methodName.value = $t('pages.transactions.deposit_action_name');
     } else if (
         !props.trx.parsedTransaction
         && props.trx.to === ZERO_ADDRESSES
-        && props.trx.value
+        && propValue.value
         && parseInt(props.trx.gasPrice as string) === 0
     ) {
         nativeTooltipText.value = $t('pages.transactions.native_withdraw_tooltip');
         methodName.value = $t('pages.transactions.withdraw_action_name');
-    } else if (!props.trx.parsedTransaction && props.trx.input === '0x' && props.trx.value) {
+    } else if (!props.trx.parsedTransaction && props.trx.input === '0x' && propValue.value) {
```

Comment:
we reference `0x` here and `0x0` on line 63, are these handling distinct cases or can we use `ZERO_ADDRESSES` for these?